src/components/ChatEntry.js

```diff
@@ -1,22 +1,57 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ id, sender, body, timeStamp, liked, toggleLike }) => {
+  //
+  const isLocal = (id) => {
+    if (!(id % 2 === 0)) {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
+  const heartColor = liked ? '❤️' : '🤍';
+
+  const handleToggleLike = () => {
+    const updatedEntry = {
+      id: id,
+      sender: sender,
+      body: body,
+      timeStamp: timeStamp,
+      liked: !liked,
+    };
+    toggleLike(updatedEntry);
+  };
 
-const ChatEntry = (props) => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
-      <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+    <div
+      className={`${isLocal(id) ? 'chat-entry local' : 'chat-entry remote'}`}
+    >
```

Comment:
👍 nice ternary for determining the correct class selector via `isLocal()`. Clever!